exporter/metric/metric.go

```diff
@@ -469,8 +469,7 @@ func gaugeToTimeSeries[N int64 | float64](point metricdata.DataPoint[N], metrics
 		ValueType:  valueType,
 		Points: []*monitoringpb.Point{{
 			Interval: &monitoringpb.TimeInterval{
-				StartTime: timestamp,
-				EndTime:   timestamp,
+				EndTime: timestamp,
```

Comment:
StartTime is not required for gauges.  This matches the collector exporter's behavior.